buf.gen.yaml

```diff
@@ -0,0 +1,19 @@
+version: v1
+plugins:
+  - remote: buf.build/protocolbuffers/plugins/go
+    out: generated/go/tdex-daemon
```

Comment:
this will create a generated folder in the root folder, it shoudl be src/api-spec etc.?

Reply:
remove